script/build.js

```diff
@@ -125,7 +125,7 @@ smith.metadata({ buildtype: options.buildtype });
 const ignore = require('metalsmith-ignore');
 const ignoreList = [];
 if (options.buildtype === 'production') {
-  ignoreList.push('education/gi-bill/post-9-11/status');
+  ignoreList.push('education/gi-bill/post-9-11/status/*');
```

Comment:
@raquelromano-gov I noticed that this app isn't currently being blocked in production. I think this change will fix that.